Cypress/cypress/integration/Parallel_Tests/Create_Workflow/preDefinedWorkflow.spec.js

```diff
@@ -0,0 +1,76 @@
+/// <reference types="Cypress" />
+import * as workflows from "../../../fixtures/Workflows.json";
+import * as user from "../../../fixtures/Users.json";
+
+describe("Testing the workflow creation wizard using PreDefined Experiments", () => {
+	before("Clearing the Cookies and deleting the Cookies", () => {
+		cy.requestLogin(user.AdminName, user.AdminPassword);
+		cy.waitForCluster("Self-Agent");
+		cy.visit("/create-workflow");
+	});
+
+	it("Running PreDefined Workflow", () => {
+		cy.chooseAgent(0);
+		cy.get("[data-cy=ControlButtons] Button").eq(0).click();
+		cy.GraphqlWait("GetPredefinedWorkflowList", "getPredefinedData");
+
+		cy.chooseWorkflow(0, 0, "@getPredefinedData");
+
+		cy.configureWorkflowSettings(
+			workflows.nonRecurringworkflowName,
+			workflows.nonRecurringworkflowDescription,
+			0
+		);
+		cy.get("[data-cy=ControlButtons] Button").eq(1).click();
+		cy.wait(1000); // Needs to be removed with frontend enhancement
```

Comment:
Post this can you also add a check to verify if the dagre graph and experiments are present here!

Reply:
Also if the Configuration stepper exists and we can go through the step and click on Finish.